system/services/presideObjects/SqlSchemaSynchronizer.cfc

```diff
@@ -190,21 +198,53 @@ component {
 			column.definitionSql = adapter.getColumnDefinitionSql( argumentCollection = args );
 			column.alterSql      = adapter.getAlterColumnSql( argumentCollection = args );
 			column.addSql        = adapter.getAddColumnSql( argumentCollection = args );
-			column.version       = Hash( column.definitionSql );
-
-			columnSql &= delim & column.definitionSql;
+			if ( _skipSync( colMeta ) ) {
+				dbColumn = QueryFilter( columnsFromDb, function( col ) {
```

Comment:
I still feel uneasy about this block of code. If we are skipping sync of a column, I'd like to see that we don't gather any information about it at all. This feels like it could be subject to failure because we still then check this column later on. 

Can this be reworked so that we explicitly _skip_ the column when looping the columns in the update logic, rather than setting the SQL to what we think it currently is?

Reply:
@DominicWatson Let me check further & try to make the changes as per your request.

Reply:
@DominicWatson  I did the below functionality to avoid property ( if dbsync=false ) 

![image](https://user-images.githubusercontent.com/532401/100190492-5e20ee80-2f14-11eb-8dc4-59e0041cb45b.png)

but preside consider as that column as deprecated and renamed with __depricated__ in DB.

Is it okay for you ?

**NOTE: **  To avoid these problem previously I've implemented the functionality to get old version of schema . 

do you have any idea related that.

Reply:
So, right now the logic in `_updateDbTable()` is:

```cfc
for( column in columnsFromDb ){
// ...
    if ( StructKeyExists( colsSql, columnName ) ) {
       // ... update column if necessary
    } else if ( !column.column_name contains "__deprecated__" ) {
       // ... deprecate the column
    }
}
```

In here, we need to add a check to see if there is a local matching property for the DB column that dbsync = false. I think the cleanest place for this would be at the start of the loop. Something like:

```
for( column in dbColumns ) {
    if ( StructKeyExists( arguments.objectProperties, column.column_name ) && _skipSync( arguments.objectProperties[ column.column_name ] ) ) {
        continue;
    }
}
```

?

Reply:
Thanks @DominicWatson  will check

Reply:
Hi @DominicWatson  , 

I've changed code as per you mentioned , please check. 

shall we change this feature in docs ? if yes means what are the file I need to look ?